setup.cfg

```diff
@@ -97,7 +97,6 @@ description = run the tests in docs
 setenv =
     PIP_DISABLE_PIP_VERSION_CHECK = 1
     {py27,pypy}: PYTHONWARNINGS=ignore:DEPRECATION::pip._internal.cli.base_command
-    DBT_PROFILES_DIR = {env:DBT_PROFILES_DIR:{toxinidir}/docs/source/_static/dbt_project}
```

Comment:
You have to add the `--profiles-dir` flag in the `pytest` call a couple lines below

Reply:
good catch https://github.com/godatadriven/pytest-dbt-core/pull/25/commits/a5420f3f1feac5d753d2cae1cd002bfa51e8862f